Styles.css

```diff
@@ -0,0 +1,85 @@
+* {
+  margin: 0%;
+  padding: 0%;
+}
+
+body {
+  width: 375px;
+  height: 6968px;
+  margin-right: 486.5px;
+  margin-left: 486.5px;
+}
```

Comment:
Please consider removing the fixed-sized dimensions on your webpage, fixed dimensions make your webpage unresponsive on certain screen sizes. When the style sheet is loaded correctly this is how the webpage renders please see the screenshot below.

## Current mobile view in browser with 
`margin-right: 486.5px` and `margin-left: 486.5px` 

<img src="https://user-images.githubusercontent.com/16392046/191865569-c21f4180-31cf-4b57-87e2-0a4f46cafc42.png" width=200px />

## Mobile view in the browser without the fixed dimensions

<img src="https://user-images.githubusercontent.com/16392046/191866010-f54fabf5-fb2f-4a64-8756-68766bcc4f6e.png" width=200px />